easybuild/easyconfigs/h/HiC-Pro/HiC-Pro.patch

```diff
@@ -0,0 +1,34 @@
+diff -ruN HiC-Pro-2.9.0/config-system.txt HiC-Pro-2.9.0-patched/config-system.txt
```

Comment:
@timeu Please include a short description of the patch above this line, and mention the author (or include a reference to where you got the patch from)

Reply:
@timeu Please remove this old (now unused) patch?